src/libs/runtime-utils.js

```diff
@@ -164,10 +169,24 @@ export const machineCost = machineType => {
 
 export const currentApp = _.flow(trimAppsOldestFirst, _.last)
 
-export const currentDataDisk = (app, galaxyDataDisks) => {
+export const currentAttachedDataDisk = (app, galaxyDataDisks) => {
   return _.find({ name: app?.diskName }, galaxyDataDisks)
 }
 
+export const getCurrentPersistentDiskName = (apps, galaxyDataDisks) => {
```

Comment:
Since this method is used only by `NewGalaxyModal`, it may be good to move it to `NewGalaxyModal.js` file.